packages/shared/src/components/checklist/SidebarOnboardingChecklistCard.tsx

```diff
@@ -0,0 +1,138 @@
+import dynamic from 'next/dynamic';
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { ChecklistCardVariant, ChecklistViewState } from '../../lib/checklist';
+import { Button, ButtonSize, ButtonVariant } from '../buttons/Button';
+import { ArrowIcon, EyeCancelIcon, MenuIcon } from '../icons';
+import ContextMenu from '../fields/ContextMenu';
+import { ContextMenu as ContextMenuIds } from '../../hooks/constants';
+import useContextMenu from '../../hooks/useContextMenu';
+import { useOnboardingChecklist } from '../../hooks';
+import { RankConfetti } from '../../svg/RankConfetti';
+import { withExperiment } from '../withExperiment';
+import { feature } from '../../lib/featureManagement';
+
+const OnboardingChecklistCard = dynamic(() =>
+  import(
+    /* webpackChunkName: "onboardingChecklistCard" */ './OnboardingChecklistCard'
+  ).then((mod) => mod.OnboardingChecklistCard),
+);
+
+export type SidebarOnboardingChecklistCardProps = {
+  className?: string;
+};
+
+const SidebarOnboardingChecklistCardComponent = ({
+  className,
+}: SidebarOnboardingChecklistCardProps): ReactElement => {
+  const { checklistView, setChecklistView, isDone } = useOnboardingChecklist();
+  const { onMenuClick: showOptionsMenu, isOpen: isOptionsOpen } =
+    useContextMenu({
+      id: ContextMenuIds.SidebarOnboardingChecklistCard,
+    });
+
+  if (checklistView === ChecklistViewState.Hidden) {
+    return null;
+  }
+
+  const isOpen = checklistView === ChecklistViewState.Open;
+
+  if (isDone) {
+    return (
+      <div
+        className={classNames(
+          'relative m-2 mt-0 flex flex-col items-center gap-2 overflow-hidden rounded-8 bg-gradient-to-t from-raw-cabbage-90 to-raw-cabbage-50 p-2',
+        )}
+      >
+        <RankConfetti className="pointer-events-none absolute bottom-0 left-0 right-0 top-0 opacity-40" />
+        <p
+          className={classNames(
+            'mb-1 text-center font-bold text-white typo-footnote',
+          )}
+        >
+          Get started like a pro
+          <br />
+          Perfectly done!
+        </p>
+        <Button
+          variant={ButtonVariant.Secondary}
+          size={ButtonSize.XSmall}
+          onClick={() => {
+            setChecklistView(ChecklistViewState.Hidden);
+          }}
+        >
+          Dismiss
+        </Button>
+      </div>
```

Comment:
This could be it's own ChecklistCompletedHeader component

Reply:
Then would it make sense that within the checklist folder to have the primitive types, and then within another folder like onboardingChecklist to have all things related to that only? this way if we extract a header for onboardingChecklist we know it's for this particular checklist. Just more a fan of not making large files with multiple components where it kind of makes sense to be it's own component - maybe not reusable anywhere else now but it's still a component for the onboarding checklist card.

Reply:
Moved it to `OnboardingChecklistOptions`